nix/pkgs/codecov-cli/default.nix

```diff
@@ -16,6 +16,7 @@ in stdenv.mkDerivation rec {
     url = "https://cli.codecov.io/v${version}/${platform}/codecov";
     hash = lib.getAttr builtins.currentSystem {
       aarch64-darwin = "sha256-CB1D8/zYF23Jes9sd6rJiadDg7nwwee9xWSYqSByAlU=";
+      x86_64-darwin = "sha256-CB1D8/zYF23Jes9sd6rJiadDg7nwwee9xWSYqSByAlU=";
```

Comment:
pretty sure this sha is wrong.
the sha of `x86_64-darwin` should be different from sha of `aarch64-darwin`

Reply:
oh my bad I just saw this comment.

>  There's no arm64 build for macOS, amd64 works on both

Reply:
yep, also I didn't just copied it, downloaded the `x86_64-darwin`  codecov pkg and ran:
```
shasum -a 256 codecov | awk '{print $1}' | xxd -r -p | base64
```